scripts/utilities.js

```diff
@@ -83,10 +83,29 @@ function shouldContinuePrompt(callback = null) {
   }
 }
 
+const packDependency = () => {
+  spawnOrFail('npm', ['install']);
+  spawnOrFail('npm', ['run build']);
+  spawnOrFail('npm', ['pack']);
+}
+
+const updateDependency = (name, path) => {
+  if(path) {
+    logger.log(`Installing ${name} from local file`);
+    spawnOrFail('npm', [`install ${path}`]);
+  }else{
+    const updatedVersion = spawnOrFail('npm', [`show ${name} version`]).trim();
```

Comment:
Instead of doing this, can we just allow passing name@version as the parameter? We already know what version we want to install when calling this method so you just need to pass `amazon-chime-sdk-js@{version}`.